static/src/stylesheets/module/_social.scss

```diff
@@ -159,6 +165,13 @@ category: Common
     }
 }
 
+.social-icon--messenger {
+    background-color: $social-messenger;
+    &:hover {
+        background-color: darken($social-messenger, 10%);
```

Comment:
`10%` is what we have for `twitter`, `whatsapp`, `gplus`, so I don't think we should be more specific for messenger here.